x11-packages/nitrogen/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=http://projects.l3ib.org/nitrogen/
+TERMUX_PKG_DESCRIPTION="Background browser and setter for X windows."
```

Comment:
Please remove '.'(period) from end. Last sentence (here i.e just one) should not have that.